addons/survey/static/src/scss/survey_survey_views.scss

```diff
@@ -146,3 +146,31 @@
 .icon_rotates {
     transform: rotate(180deg);
 }
+
+/* Style of the tiles allowing the user to load a sample survey. */
+.o_survey_sample_tile {
+    position: relative;
+    width: 150px;
+    height: 150px;
+    img {
+        max-height: 125px;
+    }
+    div {
+        display: none;
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: 100%;
+        height: 100%;
+        overflow-y: auto;
+        cursor: pointer;
+    }
+    &:hover {
```

Comment:
By the way if the text overflow it's cropped at the top:

![image](https://user-images.githubusercontent.com/41282536/143423800-cb5d0215-b93b-4f4a-a274-334d0222b292.png)

Could happen if translations are longer than the english text.